internal/versions/versions.go

```diff
@@ -228,6 +231,10 @@ func (h *handler) GetOsImage(openshiftVersion, cpuArchitecture string) (*models.
 		// Empty implies default CPU architecture
 		cpuArchitecture = common.DefaultCPUArchitecture
 	}
+	// RHCOS uses aarch64 for arm64
+	if cpuArchitecture == "arm64" {
+		cpuArchitecture = "aarch64"
+	}
```

Comment:
This function has quite an extensive set of unit tests in around https://github.com/openshift/assisted-service/blob/master/internal/versions/versions_test.go#L929-L934 and https://github.com/openshift/assisted-service/blob/master/internal/versions/versions_test.go#L403-L421

It would be very appreciated if you created a set of cases there that cover `GetOsImage` via `arm64` as well as via `aarch64` so that by reading the test I can understand the rationale behind the change.

Look e.g. at the test https://github.com/openshift/assisted-service/blob/master/internal/versions/versions_test.go#L579-L592 where we explicitly have a statement "gets X using X query" and "gets X using Y query"